api/v1beta1/common.go

```diff
@@ -162,3 +169,39 @@ type AnsibleEESpec struct {
 	// Env is a list containing the environment variables to pass to the pod
 	Env []corev1.EnvVar `json:"env,omitempty"`
 }
+
+// NovaTemplate specifies the parameters for the compute service deployment on
+// the EDPM node.
+type NovaTemplate struct {
+	// +kubebuilder:validation:Optional
```

Comment:
defaulting to cell 1 is tempting but i think we are going to see multi cell deployments become more common so long term this should be required not optional.

can address that in a separate pr but we should make this change in Q2.

Reply:
I'm OK to make CellName a required field. But I would like to do that in a separate PR to limit the amount of change here.